engineering-plans/0001-graphql-query-prefetching.md

```diff
@@ -0,0 +1,397 @@
+# PLAN-0001: GraphQL Query Prefetching
+
+<dl>
+  <dt>Author</dt>
+  <dd>David Lutterkort</dd>
+
+<dt>Implements</dt>
+<dd>No RFC - no user visible changes</dd>
+
+<dt>Engineering Plan pull request</dt>
+<dd><a href="<https://github.com/graphprotocol/rfcs/pull/2>"><https://github.com/graphprotocol/rfcs/pull/2></a></dd>
+
+<dt>Date of submission</dt>
+<dd>2019-11-27</dd>
+
+<dt>Date of approval</dt>
+<dd>2019-12-10</dd>
+
+  <dt>Approved by</dt>
+  <dd>Jannis Pohlmann, Leo Yvens</dd>
```

Comment:
I think @leoyvens name is Leo(nardo) Yvens Schwarzstein. 😉 Leo: Would you prefer to have the complete name in these RFCs or just a part of it?

Reply:
I think the only ones who would remember how to spell 'Schwarzstein' are either me (most of the time) and those who speak german 😄

Leo Yvens is fine since that's also my handle.

Reply:
I wasn't sure what to put for you, but glad I guessed right. Not that I'd have a problem with saying Schwarzstein: Schwarzstein, Schwarzstein, Schwarzstein (one day you have to tell me how that's pronounced in Portuguese) ;)

Reply:
@lutter In Portuguese it's pronounced Schwarzenegger.

Reply:
Hahahaha